app/assets/stylesheets/splash_screen.css

```diff
@@ -0,0 +1,108 @@
+.header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3rem;
+}
+
+.checkpoints {
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.checkpoint {
+  margin-bottom: 1rem;
+  color: #434b54;
+}
+
+.my_wallet {
+  margin-bottom: 2rem;
+}
+
+.signup-form {
+  text-align: center;
+}
+
+.btn-primary {
+  background-color: #5fb523;
+  color: #fff;
+  padding: 1.5rem 2rem;
+  font-size: 1rem;
+  border: none;
+  cursor: pointer;
+  transition: background-color 0.3s ease-in-out;
+}
+
+.btn-primary:hover {
+  background-color: #4a971f; /* Slightly darker shade on hover */
```

Comment:
Your app is working well, and with awesome UI 👌 However, it will be a good idea if we can add a few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:

- [ ] Let's ensure the `Add Category, Create Category, New Transaction and similar buttons should conform to the stecified typography,  color (green with white rectangular inner border) and layout (is always sticky to the bottom of the screen) like can be seen in the template design.
- [ ] Typography and the general layout of the app design should be similar to the original design.

**Your design:**
**Login:**        |  **Your categories page:**        |  **Your transaction page**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Nessrine88/My_Wallet/assets/12745474/53c137fe-a7b5-4e9c-9489-784b86bd459e"> | <img width="800" alt="Home Page" src="https://github.com/Nessrine88/My_Wallet/assets/12745474/9062235d-8a9f-4bce-83ad-63d7134e31b9"> | <img width="800" alt="Home Page" src="https://github.com/Nessrine88/My_Wallet/assets/12745474/6a7f1752-8951-4feb-b21f-7b1ec38aa722">


**Expected:**

**Login:**        |  **Categories page:**        |  **Transaction page**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Nessrine88/My_Wallet/assets/12745474/2c637a0a-3023-4bc9-a015-ca87592abe0b"> | <img width="800" alt="Home Page" src="https://github.com/chudisoft/budget_manager/assets/12745474/c091dc96-c486-4d00-84a9-9099d2cd2e7c"> | <img width="800" alt="Home Page" src="https://github.com/Nessrine88/My_Wallet/assets/12745474/11b72d3b-52fa-4c29-b009-ed64872db920">